.gitlab/ci/.gitlab-ci.bucket-upload.yml

```diff
@@ -221,18 +221,18 @@ install-packs-in-server-master:
     - CLOUD_SERVERS_PATH=$(cat $CLOUD_SERVERS_FILE)
     - echo ${CLOUD_API_KEYS} > "cloud_api_keys.json"
 
+    - section_start "Clean Machine"
+    - ./Tests/scripts/uninstall_packs_and_reset_bucket_cloud.sh || EXIT_CODE=$?
```

Comment:
don't fail on errors and proceed regardless, but still remember to fail the build.